### PR TITLE
Fix issue participants

### PR DESCRIPTION
**Pull request by silverwind**
Screen Shot 2020-12-07 at 22 12 51

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: templates/repo/issue/view_content/sidebar.tmpl [hunk 1]
```diff
@@ -292,12 +292,10 @@
 
 		<div class="ui participants">
```
Comment (lunny) [inline templates/repo/issue/view_content/sidebar.tmpl]: `.ui.participants`  still is referenced here?

Comment (silverwind) [reply templates/repo/issue/view_content/sidebar.tmpl]: I've refactored it to remove this wrapper, but I guess it makes sense to keep the `participants` class for consistency reason only as the other lists also all have such a name:

image

Comment (silverwind) [reply templates/repo/issue/view_content/sidebar.tmpl]: Actually, decided to remove it.

**Review by 6543 — APPROVED** [review]

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**